### PR TITLE
Faraz/lp 1387 integrate onboarding

### PR DESCRIPTION
**Pull request by ahmed-faraz46**

### CODE REVIEW

File: lms/djangoapps/onboarding/forms.py [hunk 1]
```diff
@@ -502,15 +502,16 @@ def clean_country(self):
 
     def clean_url(self):
         is_org_url_exist = int(self.data.get('is_org_url_exist')) if self.data.get('is_org_url_exist') else None
-        organization_website = self.cleaned_data.get('url', '').replace('http://', 'https://', 1)
+        org_website = self.cleaned_data.get('url', '')
+        org_website = org_website and self.cleaned_data.get('url', '').replace('http://', 'https://', 1)
```
Comment (emraan004) [inline lms/djangoapps/onboarding/forms.py]: Let's re-use `org_website ` variable in line 506

File: lms/djangoapps/philu_overrides/migrations/0001_initial.py [hunk 2]
```diff
@@ -11,9 +11,7 @@ class Migration(migrations.Migration):
 
     def add_history_columns(apps, schema_editor):
         cursor = connection.cursor()
-        query = 'ALTER TABLE enterprise_historicalenterprisecustomer ADD start_date DATETIME NULL'
-        cursor.execute(query)
-        query = 'ALTER TABLE enterprise_historicalenterprisecustomer ADD end_date DATETIME NULL'
+        query = 'ALTER TABLE enterprise_historicalenterprisecustomer ADD start_date DATETIME NULL, end_date DATETIME NULL;'
```
Comment (emraan004) [inline lms/djangoapps/philu_overrides/migrations/0001_initial.py]: Let's double check it works

File: lms/djangoapps/philu_overrides/views.py [hunk 3]
```diff
@@ -79,9 +79,8 @@ def login_and_registration_form(request, initial_mode="login", org_name=None, ad
 
     """
     # Determine the URL to redirect to following login/registration/third_party_auth
-    from openedx.features.student_account.views import local_server_get, _get_form_descriptions, \
-        _third_party_auth_context
-    from openedx.core.djangoapps.user_authn.views.login_form import _external_auth_intercept
+    from openedx.features.student_account.views import local_server_get
+    from openedx.core.djangoapps.user_authn.views.login_form import _external_auth_intercept, _get_form_descriptions, _third_party_auth_context
```
Comment (emraan004) [inline lms/djangoapps/philu_overrides/views.py]: Let's apply lint rule

File: openedx/features/student_account/views.py [hunk 4]
```diff
@@ -73,7 +73,7 @@ def _get_form_descriptions(request):
     """
     return {
         'login': local_server_get('/user_api/v1/account/login_session/', request.session),
-        'registration': local_server_get('/user_api/v2/account/registration/', request.session),
+        'registration': local_server_get('/user_api/v1/account/registration/', request.session),
```
Comment (emraan004) [inline openedx/features/student_account/views.py]: Let's verify this if v1 should be replaced with v2

File: lms/djangoapps/philu_overrides/migrations/0001_initial.py (hunk 2 above)
Comment (ahmed-faraz46) [reply lms/djangoapps/philu_overrides/migrations/0001_initial.py]: this was a wrong, updated to

**Comment by ahmed-faraz46** [comment]
@emraan004 requested changes are made, please reverify. I've added a separate ticket to move `RegisterView.js` in `pilu-edx-theme`
https://philanthropyu.atlassian.net/browse/LP-1551

**Pull request merged**